docs/_source/tutorials-examples/population-synthesis/evolve_single_binaries.ipynb

```diff
@@ -0,0 +1,496 @@
+    "data_path = f'{PATH_TO_POSYDON_DATA}/POSYDON_data/tutorial/population-synthesis/examples/'\n",
+    "\n",
+    "pop = Population('BBH_contact.h5')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 143,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Because the original population contains two extre columns. These have to be defined manually here.\n",
+    "# Otherwise, the reseting of the binaries fails.\n",
+    "BINARY = BinaryStar.from_df(pop.history[0],\n",
+    "                            extra_columns={'step_names':'string','step_times':'float'})\n",
+    "\n",
+    "# you have to set the simulation properties for the binary\n",
+    "BINARY.properties = sim_prop\n",
+    "\n",
+    "# Set the natal kick arrays from the oneline to get the exact same evolution as the original binary\n",
+    "BINARY.star_1.natal_kick_array = pop.oneline[0][['S1_natal_kick_array_0', 'S1_natal_kick_array_1', 'S1_natal_kick_array_2', 'S1_natal_kick_array_3']].values.flatten()\n",
+    "BINARY.star_2.natal_kick_array = pop.oneline[0][['S2_natal_kick_array_0', 'S2_natal_kick_array_1', 'S2_natal_kick_array_2', 'S2_natal_kick_array_3']].values.flatten()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.to_df(extra_columns={'step_names':'string'})[col]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 146,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.restore()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.evolve()\n",
+    "BINARY.to_df(extra_columns={'step_names':'string'})[col]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evolving a Binary Star starting from an arbitrary state"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to evolve a custom binary star, you can do so by crafting yourself the `BinaryStar` object and providing the `SimulationProperties` object. For example, let's evolve a neutron star with a low mass helium star in Roche lobe overflow.\n",
+    "And start a binary in the detached step, which requires a bit of additional input data.\n",
+    "\n",
+    "We will use the standard SimulationProperties, loaded earlier in this notebook.\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-warning\"><b>Caution!</b> \n",
+    "\n",
+    "While you can evolve a binary from an arbitrary state, you will need to provide data on the internal structure of the star, if you're starting in the detached step.\n",
+    "Otherwise, the matching to a single star model will not work!\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we intiialise the binary to be tight and Roche lobe overflowing.\n",
```

Comment:
I guess, you should mention, that it the orbit should be circular, or is the circularization not usually done in the previous step to RLO.

Reply:
I state this.

Reply:
Where? I haven't seen any information about circular orbits in this notebook.

By the way another typo: "intiialise" -> "initialise"/"initialize"

Reply:
I've added it in the next commit. additionally spelling was changed